package.json

```diff
@@ -46,6 +46,21 @@
 		"validate": "npm ls",
 		"version": "git add package.json && git commit -m \"Bump package version\""
 	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "lint-staged"
+		}
+	},
+	"lint-staged": {
+		"*.{js,jsx}": [
+			"npm run lint:fix",
+			"git add"
+		],
+		"*.{js,jsx,ts,tsx,css,scss}": [
```

Comment:
I don't think we need to support typescript files. I would just remove `ts and tsx`.